### DISCUSSION TITLE
information

### DISCUSSION THREAD

**Discussion by philsludge** [community/community#81882] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I may be in over my head.  I am using the app "Test Netspeed"  I have a number of "firesticks", and am paying for Spectrum Internet.
I use the graph/chart function and it shows what I already know:  Spectrum internet is lousy, or there is a problem with the service OUTSIDE of my house.  I will call for a tech to come after New Years day.  I purchased a brand new Modem/Router about a week to 10 days ago and It help quite a bit but lousy is still lousy.  Spectrum will never let any speed test but their own being used to tell what is going on.   Their test is an average and it is always fairly high, but does not show what is happening at any period in time. 
My question is about the graph.   What are the values on the graph/chart:  minimum, to maximum, time line shown and any other values that can be shown?

**Comment by Akash1134** [comment]
Hi @philsludge ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
